generated-dockerfiles/centos7-base.Dockerfile

```diff
@@ -14,7 +14,7 @@ ARG FROM_IMAGE=gpuci/rapidsai
 FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
 
 ARG DASK_XGBOOST_VER=0.2*
-ARG RAPIDS_VER=0.15*
+ARG RAPIDS_VER
```

Comment:
Shouldn't this have a value? That way if you do just `docker build` without args it will build with all of the default values?

Reply:
> Shouldn't this have a value? That way if you do just `docker build` without args it will build with all of the default values?

It will use the value from before the `FROM` statement [here](https://github.com/rapidsai/docker/blob/6a44f90ecdc561f1b67cfff21e4ae35c5108e2d6/generated-dockerfiles/centos7-devel.Dockerfile#L13). 
(src: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact).

I was able to confirm this locally with the similar example below:

```dockerfile
ARG ALP_VER=3

FROM alpine:${ALP_VER}

ARG ALP_VER

RUN echo "${ALP_VER}"

```

I ran both `docker build .` and `docker build --build-arg "ALP_VER=3.12" .` and got the expected values.

Reply:
Got it my mistake I see you're defining this to use it again later